src/core/utils.py

```diff
@@ -0,0 +1,22 @@
+from src.core.common_path import CommonPaths
```

Comment:
@vasylboliuk please don't forget to add docstrings at the module and class level. Could you please update it through your code